### PR TITLE
added post to readme

### PR DESCRIPTION
**Pull request by ogsmith**

### CODE REVIEW

**Review by remstone7 — COMMENTED** [review]
Hate to do this, but looks like the previous pr to add a post request is missing docstrings, which looks like it should actually be a bool not a str based on: https://github.com/klaviyo/php-klaviyo/blob/master/src/PublicAPI.php#L41. mind just updating this?

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ $event = new KlaviyoEvent(
     )
 );
 
-$client->publicAPI->track( $event );
+$client->publicAPI->track( $event, $post='POST' );
```
Comment (remstone7) [inline README.md:62]: these should also be true?

Comment (ogsmith) [reply README.md]: Edited, my bad

File: README.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ $profile = new KlaviyoProfile(
     )
 );
 
-$client->publicAPI->identify( $profile );
+$client->publicAPI->identify( $profile, $post=true );
```
Comment (remstone7) [inline README.md:78]: one last thing, unless someone is on php 8, named arguments aren't supported so we can just do:
```suggestion
$client->publicAPI->identify( $profile, true );
```

File: README.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ $event = new KlaviyoEvent(
     )
 );
 
-$client->publicAPI->track( $event );
+$client->publicAPI->track( $event, $post=true );
```
Comment (remstone7) [inline README.md:62]: can you update these as well, then we're good to merge

**Pull request merged**